src/Tizen.NUI.Components/Controls/Loading.cs

```diff
@@ -1,5 +1,5 @@
 ﻿/*
- * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
```

Comment:
If it's not a new file, then it would be better to put list or range:  "2021, 2022" OR "2021-2022"

Reply:
Thanks for reviewing this PR.
Yes, it's not a new file. However, as you know, NUI is csharp-binding of native DALi, and DALi has a rule that updates the copyright year when the file is updated. So, as far as I know, NUI also follows this rule.